yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/profile/request/attribute/derivation/AgeVerification.java

```diff
@@ -0,0 +1,26 @@
+package com.yoti.api.client.sandbox.profile.request.attribute.derivation;
+
+import com.yoti.api.client.Date;
+import com.yoti.api.client.sandbox.profile.request.attribute.Attribute;
+import com.yoti.api.client.sandbox.profile.request.attribute.ProfileAttributeName;
+
+public class AgeVerification {
+
+    private Date dateOfBirth;
+    private AgeDerivationSupported ageDerivationSupported;
+    private String ageCheckOption;
+
+    public AgeVerification(Date dateOfBirth, AgeDerivationSupported ageDerivationSupported, String ageCheckOption) {
+        this.dateOfBirth = dateOfBirth;
+        this.ageDerivationSupported = ageDerivationSupported;
+        this.ageCheckOption = ageCheckOption;
```

Comment:
what is this for?

Reply:
This is used in the `SandboxHumanProfile` for giving to the developer a good way to wrap an `AgeVerification-related` attribute during the building process.